exhibitor-core/src/main/java/com/netflix/exhibitor/core/azure/PropertyBasedAzureCredential.java

```diff
@@ -0,0 +1,51 @@
+package com.netflix.exhibitor.core.azure;
+
+import org.apache.curator.utils.CloseableUtils;
+
+import java.io.*;
+import java.util.Properties;
+
+public class PropertyBasedAzureCredential implements AzureCredential {
+    private final String accountName;
+    private final String accountKey;
+
+    public static final String PROPERTY_AZURE_ACCOUNT_NAME = "com.netflix.exhibitor.azure.account-name";
+    public static final String PROPERTY_AZURE_ACCOUNT_KEY = "com.netflix.exhibitor.azure.account-key";
+
+    public PropertyBasedAzureCredential(File propertiesFile) throws IOException
```

Comment:
I'm almost sure we're following Google (Sun) Java style - this is C++-style
please make sure opening braces are on the same line as the method definition